Windows CI jobs failing due to meson

## Bug Report

https://github.com/versatica/mediasoup/actions/runs/7395922187/job/20120052331?pr=1291

In summary:

```
"C:\hostedtoolcache\windows\Python\3.9.13\x64\python3.exe" -m pip install --upgrade --no-user --target="D:\a\mediasoup\mediasoup\worker/out/pip_meson_ninja"  meson==1.3.0 ninja==1.10.2.4
Collecting meson==1.3.0

  Using cached meson-1.3.0-py3-none-any.whl.metadata (1.8 kB)

Collecting ninja==1.10.2.4

  Using cached ninja-1.10.2.4-py2.py3-none-win_amd64.whl (293 kB)

Using cached meson-1.3.0-py3-none-any.whl (976 kB)

Installing collected packages: ninja, meson

Successfully installed meson-1.3.0 ninja-1.10.2.4

cd D:\a\mediasoup\mediasoup\worker && "D:\a\mediasoup\mediasoup\worker/out/pip_meson_ninja/bin/meson" setup --prefix "D:\a\mediasoup\mediasoup\worker/out/Release" --bindir "" --libdir "" --buildtype release -Db_ndebug=true -Db_pie=true -Db_staticpic=true --vsenv "D:\a\mediasoup\mediasoup\worker/out/Release/build"
Directory already configured.


Just run your build command (e.g. ninja) and Meson will regenerate as necessary.

Run "meson setup --reconfigure to force Meson to regenerate.


If build failures persist, run "meson setup --wipe" to rebuild from scratch

using the same options as passed when configuring the build.


ERROR: Unknown options: "b_pie, b_staticpic"

npm-scripts.mjs [ERROR] [prepare] executeCmd() failed, exiting: Error: Command failed: "python3" -m invoke -r worker flatc
npm ERR! code 1
npm ERR! path D:\a\mediasoup\mediasoup
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node npm-scripts.mjs prepare
```


We may want to happily blame this recent merged PR https://github.com/versatica/mediasoup/pull/1275 but it wouldn't be fair since it did pass all CI jobs and they also passed in more recent PRs and changes in v3 branch. This is something new. New meson version in Windows?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@
 
 * Avoid modification of user input data ([PR #1285](https://github.com/versatica/mediasoup/pull/1285)).
 * `ListenInfo`: Add transport socket flags ([PR #1291](https://github.com/versatica/mediasoup/pull/1291)).
+* Meson: Remove `-Db_pie=true -Db_staticpic=true` args ([PR #1293](https://github.com/versatica/mediasoup/pull/1293)).
 
 
 ### 3.13.13

--- a/worker/tasks.py
+++ b/worker/tasks.py
@@ -164,6 +164,15 @@ def setup(ctx):
                 shell=SHELL
             );
 
+    # TMP to see meson build args available in current host:
+    with ctx.cd(WORKER_DIR):
+        ctx.run(
+            f'"{MESON}" configure',
+            echo=True,
+            pty=PTY_SUPPORTED,
+            shell=SHELL
+        );
+
 
 @task
 def clean(ctx): # pylint: disable=unused-argument

--- a/worker/tasks.py
+++ b/worker/tasks.py
@@ -142,7 +142,7 @@ def setup(ctx):
     if MEDIASOUP_BUILDTYPE == 'Release':
         with ctx.cd(WORKER_DIR):
             ctx.run(
-                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype release -Db_ndebug=true -Db_pie=true -Db_staticpic=true {MESON_ARGS} "{BUILD_DIR}"',
+                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype release -Db_ndebug=true {MESON_ARGS} "{BUILD_DIR}"',
                 echo=True,
                 pty=PTY_SUPPORTED,
                 shell=SHELL
@@ -150,7 +150,7 @@ def setup(ctx):
     elif MEDIASOUP_BUILDTYPE == 'Debug':
         with ctx.cd(WORKER_DIR):
             ctx.run(
-                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype debug -Db_pie=true -Db_staticpic=true {MESON_ARGS} "{BUILD_DIR}"',
+                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype debug {MESON_ARGS} "{BUILD_DIR}"',
                 echo=True,
                 pty=PTY_SUPPORTED,
                 shell=SHELL
@@ -158,7 +158,7 @@ def setup(ctx):
     else:
         with ctx.cd(WORKER_DIR):
             ctx.run(
-                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype {MEDIASOUP_BUILDTYPE} -Db_ndebug=if-release -Db_pie=true -Db_staticpic=true {MESON_ARGS} "{BUILD_DIR}"',
+                f'"{MESON}" setup --prefix "{MEDIASOUP_INSTALL_DIR}" --bindir "" --libdir "" --buildtype {MEDIASOUP_BUILDTYPE} -Db_ndebug=if-release {MESON_ARGS} "{BUILD_DIR}"',
                 echo=True,
                 pty=PTY_SUPPORTED,
                 shell=SHELL

--- a/worker/tasks.py
+++ b/worker/tasks.py
@@ -164,15 +164,6 @@ def setup(ctx):
                 shell=SHELL
             );
 
-    # TMP to see meson build args available in current host:
-    with ctx.cd(WORKER_DIR):
-        ctx.run(
-            f'"{MESON}" configure',
-            echo=True,
-            pty=PTY_SUPPORTED,
-            shell=SHELL
-        );
-
 
 @task
 def clean(ctx): # pylint: disable=unused-argument

